resources/Help/Microsoft.Windows.Setting.Time/TimeZone.md

```diff
@@ -0,0 +1,45 @@
+---
+external help file: Microsoft.Windows.Setting.Time.psm1-Help.xml
+Module Name: Microsoft.Windows.Setting.Time
+ms.date: 05/11/2024
+online version:
+schema: 2.0.0
+title: Time
+---
+
+# Time
+
+## SYNOPSIS
+
+This `Time` DSC Resource allows you to manage the time zone, automatic time zone update, and system tray date/time visibility settings on a Windows machine.
+
+## DESCRIPTION
+
+This `Time` DSC Resource allows you to manage the time zone, automatic time zone update, and system tray date/time visibility settings on a Windows machine.
```

Comment:
```suggestion
title: TimeZone
---

# TimeZone

## SYNOPSIS

This `TimeZone` DSC Resource allows you to manage the time zone, automatic time zone update, and system tray date/time visibility settings on a Windows machine.

## DESCRIPTION

This `TimeZone` DSC Resource allows you to manage the time zone, automatic time zone update, and system tray date/time visibility settings on a Windows machine.
```

Probably need to update the descriptions as well since this only relates to `TimeZone`